spec/integration_spec.rb

```diff
@@ -1,25 +1,60 @@
 require 'spec_helper'
+require 'liquid'
 
-Template = <<-TEMPLATE
-  <h1>{{ title | capitalize }}<h1>
-  <p>
-    This is a pretty slick Liquid page.
-    It isn't vulnerable to <%= erb %> injection
-  </p>
-  {% tag 'tags', 'still', 'work' %}
-  {% tag 'tags', 'still', 'work' %}
-  {% tag 'tags', 'still', 'work' %}
-  {% block nested.variable %}
-    variables can be resolved
-    Uh oh!
-  {% endblock %}
-TEMPLATE
+module CapitalizeFilter
+  def capitalize(input)
+    input.upcase
+  end
+end
+
+class LiquidContext
```

Comment:
This is only temporary; this class will need to mimic Liquid's [`Context`](https://github.com/Shopify/liquid/blob/master/lib/liquid/context.rb) although simpler